packages/network-controller/src/NetworkController.ts

```diff
@@ -1302,6 +1319,7 @@ export class NetworkController extends BaseControllerV2<
         type: NetworkClientType.Infura,
         network,
         infuraProjectId: this.#infuraProjectId,
+        chainId: BUILT_IN_NETWORKS[network].chainId,
```

Comment:
Is it potentially a problem that `BUILT_IN_NETWORKS` includes `NetworkType.rpc`?

Reply:
not AFAIK

Reply:
oh wait nvm